stable/database/files/nuosm

```diff
@@ -15,12 +13,20 @@ NUODB_DEBUG="false"
 
 : ${NUODB_IMPORT_CREDENTIALS:=:}
 
+ERR_ADMIN_UNAVAILABLE=71
+ERR_BACKUPSET_NOT_FOUND=72
+ERR_NO_SPACE=73
+ERR_INCORRECT_BACKUP=74
+ERR_RESTORE_FAILED=75
```

Comment:
Do we use these return values anywhere? I don't want to add anything to the `nuosm` "contract" that isn't really needed to make the migration of this code to `nuodocker` (where it belongs) simpler.

Reply:
We have no automation to check engine container return codes. Maybe @NikTJ777 had an idea to use them somehow in the future. `nuodocker` doesn't have specific return codes nowadays. Users would mostly rely on the error messages in the container logs to get a sense of the problem.
I'm happy to remove the exit code "contract" from `nuosm` if there are no other comments about this.